website/static/site.css

```diff
@@ -14,25 +14,33 @@
   limitations under the License.
 **/
 
+:root {
+  --mdc-theme-primary: #306DFD;
+}
+
+th, 
+td, 
+ul li, 
 body {
-  margin: 0;
-  color: #505050;
-  font-family: Roboto, sans-serif;
-  font-display: swap;
+  font-family: 'Google Sans', sans-serif !important;
```

Comment:
Is it possible to avoid `!important` here?

Reply:
Not ideal for now, but would like to still apply it for now. Added this in order to make sure the donations table picked this up. I wanted to avoid adding too many css lines to make sure the font was being applied.

Reply:
OK - can we leave a `TODO` comment linking to a GitHub issue to remove it?

(If and when we deprecate the Flask frontend, we can close that issue + remove the TODO.)